extension/src/NABfunctions.ts

```diff
@@ -994,3 +1059,9 @@ export async function importDtsTranslations(): Promise<void> {
 
   console.log("Done: importDtsTranslations");
 }
+
+interface IExportOptions {
+  columns: string[];
+  filter: string;
```

Comment:
And why not an Enum here as well? instead of having hardcoded string values in several places...